README.md

```diff
@@ -121,6 +121,13 @@ DATABASES = {
 `OPTIONS` is an optional dictionary of parameters that will be passed to
 [`MongoClient`](https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html).
 
+Alternatively, you can use a MongoDB URI with `django_mongodb.parse(uri)`:
```

Comment:
Let's have this be the standard rather than the alternative.

Reply:
Done in 0484032244d8f905cdc12c904ed6c308b67fe6b5!